editor.planx.uk/src/@planx/components/Result/Public/index.tsx

```diff
@@ -39,26 +38,15 @@ interface Response {
   hidden: boolean;
 }
 
-const useClasses = makeStyles((theme) => ({
-  readMore: {
-    color: theme.palette.text.primary,
-    textDecoration: "underline",
-  },
-  disclaimerContent: {
-    marginTop: theme.spacing(1),
-    marginBottom: theme.spacing(1),
-    color: theme.palette.text.primary,
-  },
-  disclaimerHeading: {
-    marginRight: theme.spacing(1),
-    color: theme.palette.text.primary,
-  },
-  button: {
-    color: theme.palette.text.primary,
-    padding: theme.spacing(0.5),
-  },
+const DisclaimerContent = styled(Typography)(({ theme }) => ({
+  marginTop: theme.spacing(1),
+  marginBottom: theme.spacing(1),
 }));
 
+const DisclaimerHeading = styled(Typography)(({ theme }) => ({
+  marginRight: theme.spacing(1),
+})) as typeof Typography;
```

Comment:
Casting here is currently required when a styled component has the `component` prop.

Docs: https://mui.com/material-ui/guides/typescript/#complications-with-the-component-prop